WebDevStudios/wd_s
# Theme/acf color picker customizations

Closes #328

### DESCRIPTION ###
This adds some customization to the ACF Color Picker. Now, we can customize the swatches available in the ACF Color Picker to match brand colors for each project while also hiding the Iris color picker. This will keep from allowing every imaginable color under the sun from being chosen and restrict a user to a predefined set of colors.

### SCREENSHOTS ###
![](https://dl.dropbox.com/s/62qhfohi3cyjumk/Screenshot%202017-10-03%2022.17.40.jpg?dl=0)

### OTHER ###

### STEPS TO VERIFY ###
Check out the branch and look at the color pickers for existing blocks as well as color pickers on newly added blocks.

Update the JS in `/assets/scripts/admin-acf.js` to add a new set of custom hex codes, then see them updated in the dashboard when editing a page/post.

### DOCUMENTATION ###
We may want to update the wiki with some more details on this, but also the rest of the ACF fields...  unless we're not concerned with the nitty gritty of field setup in the wiki?  Currently, there aren't details on the field setup and usage in the wiki so if we do want to add documentation for this update we should consider doing so for all of the ACF Content Blocks.
